src/zstd.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := zstd
+$(PKG)_WEBSITE  := https://github.com/facebook/$(PKG)
```

Comment:
Avoid $(PKG) in website - it's easier to copy or navigate to a literal:
`$(PKG)_WEBSITE  := https://github.com/facebook/ztsd`